build/scripts/build.ps1

```diff
@@ -115,11 +115,12 @@ function Process-Arguments() {
 }
 
 function Run-MSBuild([string]$buildArgs = "", [string]$logFile = "", [switch]$parallel = $true) {
-    # Because we override the C#/VB toolset to build against our LKG package, it is important
-    # that we do not reuse MSBuild nodes from other jobs/builds on the machine. Otherwise,
-    # we'll run into issues such as https://github.com/dotnet/roslyn/issues/6211.
+    if (!$dotnet) {
+        $dotnet, $sdkDir = Ensure-SdkInPathAndData
+    }
+
     # MSBuildAdditionalCommandLineArgs=
-    $args = "/p:TreatWarningsAsErrors=true /warnaserror /nologo /nodeReuse:false /consoleloggerparameters:Verbosity=minimal;summary /p:Configuration=$buildConfiguration";
```

Comment:
Please ensure `nodeReuse:false` is set for Windows.

Reply:
nevermind, I see where you set it below.

Reply:
Yep - the trouble is that the .net CLI does not recognize `/nodeReuse:false` and errors when it sees it.